content/docs/components/serving/overview.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "Overview"
+description = "Model Serving Overview"
+weight = 1
++++
+
+Kubeflow provides two supported open source model serving systems that allow multi-framework model serving: KFServing and Seldon Core. You should choose the framework that best supports your model serving requirements. If you only serve  Tensorflow models using Tensorflow Serving is an option although both KFServing and Seldon support Tensorflow. A rough comparison between KFServing and Seldon Core is shown below:
+
+| Feature        | sub-feature    | KFServing | Seldon |
+|----------------|----------------|  :--:     |  :--:  |
+| Framework      | Tensorflow     | [x](https://github.com/kubeflow/kfserving/tree/master/docs/samples/tensorflow) | [x](https://docs.seldon.io/projects/seldon-core/en/latest/servers/tensorflow.html)  |
```

Comment:
On first reading, I found the "x" misleading as I thought it meant "not supported". I also didn't realize that there was a link to an example.

Recommendation: Change "x" to "yes". Perhaps also add some text above or below the table saying that the table provides a link to an example where possible.